packages/app-project/next.config.js

```diff
@@ -119,7 +116,12 @@ const nextConfig = {
 module.exports = million.next(
   withSentryConfig(nextConfig, {
     org: 'zooniverse-27',
-    project: 'fem-app-project'
+    project: 'fem-app-project',
+    sourcemaps: {
+      disable: true,
+    },
+    telemetry: false,
+    authToken: process.env.SENTRY_AUTH_TOKEN
```

Comment:
It looks like this was not required previously, but is with these changes, it looks like per https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/#extend-your-nextjs-configuration .